### DISCUSSION TITLE
Prusa Mini Input Shaper

### DISCUSSION THREAD

**Discussion by ghost** [OrcaSlicer/OrcaSlicer#2491] [Ideas]
Good day,

Is there any plan in integrating the Prusa Mini Input Shaper? I've noticed that there is a profile for the Prusa MK4 Input Shaper and the normal Prusa Mini but none for Prusa Mini Input Shaper. It has been implemented in Prusa Slicer. Hopefully, it will be added to Orca!

Thank you very much for your consideration and I am really enjoying Orca.

**Comment by m-klecka** [comment]
You can probably just copy the start and end gcodes from prusaslicer profile to OrcaSlicer, maybe some other values like speeds and such, but i haven't found any difference that would matter that much other than this. The input shaper values are coded to the mini's firmware, slicer just checks for them in the gcode.
But i may be mistaken, haven't tried this yet:)

**Comment by Teketa** [comment]
Would also love this!

**Comment by jerrytito** [comment]
Hola como estan, hice lo que  comento el compañero kleca, al principio me tiro error
despues cambien todo los g-code( cambio de capa y de mas, los copie de prusa slicer 
te tira un erron el la linea de cambio de capa que es el modo espiral o algo asi (es la tercera linea en cambio de capas)
despues de eso me permitio generar el g-code con input
espero qeu les sirva
